quickannotator/db/__init__.py

```diff
@@ -102,24 +105,24 @@ class Tile(db.Model):
     
 
 class Annotation(db.Model):
-    """Each table will follow this naming convention: {image_id}_{annotation_class_id}_{gt/pred}"""
+    """Each table will follow this naming convention: annotation_{image_id}_{annotation_class_id}_{gt/pred}"""
 
     # primary key
     id = Column(Integer, primary_key=True, autoincrement=True)
 
-    # foreign keys (these are redundant because the table name encodes the same information. But good for querying and future proofing)
+    # foreign keys (redundant due to table naming but useful for queries and future proofing)
     image_id = Column(Integer, ForeignKey('image.id'), nullable=True, default=None)
     annotation_class_id = Column(Integer, ForeignKey('annotation_class.id'), nullable=True, default=None)
     tile_id = Column(Integer, ForeignKey('tile.tile_id'), nullable=True, default=None)
-    
 
     # columns
     isgt = Column(Boolean, nullable=True, default=None)
-    centroid = Column(Geometry('POINT'))
+    centroid = Column(Geometry('POINT', srid=4326))  # Stored as geometry
     area = Column(Float)
-    polygon = Column(Geometry('POLYGON'))
+    polygon = Column(Geometry('POLYGON', srid=4326))  # Stored as geometry
```

Comment:
lets keep this srid in mind - i had some serious issues previously. if you e.g., do a query without setting SRID i think it fails silently because it basically says "nope, nothing in the same space as this". i think postgis allows for leaving blank (defaults to -1 i believe) which is beautiful - i think spatialite is more particular